include/device/PowerSensor.h

```diff
@@ -13,15 +13,19 @@ struct PowerInfo {
 };
 
 class PowerSensor : public Sensor {
+
 public:
-    static void init();
-    static float getShounVoltage();
-    static float getBusVoltage();
-    static float getCurrent();
-    static float getPower();
-    static float getLoadVoltage();
-    static PowerInfo getPowerInfo();
+    PowerSensor();
+    float getShounVoltage();
```

Comment:
Shouldn't be ShuntVoltage? Tbh I don't know what is this I have just google'd it